server/src/main/resources/application.example.yml

```diff
@@ -127,7 +127,7 @@ seata:
   server:
     raft:
       group: default
-      cluster:
+      server-addr:
```

Comment:
请直接把application.example.yml中的raft配置全部删除，因为在application.raft.example.yml中已经有相关配置了
Please delete all the raft configurations in application.example.yml directly, because there are already relevant configurations in application.raft.example.yml.